freqtrade/tests/optimize/test_backtesting.py

```diff
@@ -139,18 +127,24 @@ def test_backtest2(default_conf, mocker):
     mocker.patch.dict('freqtrade.main._CONF', default_conf)
     data = optimize.load_data(ticker_interval=5, pairs=['BTC_ETH'])
     results = backtest(default_conf['stake_amount'], optimize.preprocess(data), 10, True)
-    num_resutls = len(results)
-    assert num_resutls > 0
+    num_results = len(results)
+    assert num_results > 0
 
 
 def test_processed(default_conf, mocker):
     mocker.patch.dict('freqtrade.main._CONF', default_conf)
     data = load_data_test('raise')
-    assert optimize.preprocess(data)
+    x = optimize.preprocess(data)
```

Comment:
In trivial code it isn't so bad, but I would still use for example `preprocessed` or `ticker` instead of `x` as variable name.

Reply:
A good named variable is always better for reading it